docs/docs/user-guide/using-own-identity.md

```diff
@@ -80,6 +80,13 @@ You will be presented with the `Configured permissions` section again but this t
 
 [![SharePoint Online Configured permissions](../images/using-own-identity/spo-configured-permissions.png)](../images/using-own-identity/spo-configured-permissions.png)
 
+!!! attention
+    Note that when executing tenant level SharePoint Online commands, the CLI will attempt to autodiscover your tenant URL. 
+    
+    It will first check for the presence of an environment variable containing the tenant URL, which can be set by using [`m365 spo set`](../../cmd/spo/spo-set/) command.
+    
+    If this cannot be found, a request will be made to the Microsoft Graph. For this request to be succesful, you must ensure that you have at least the `Sites.Read.All` permission granted to your application.
```

Comment:
Typo in successful

Reply:
I have to laugh as I always get this word wrong 🤦‍♂️ 

If only we had an automated process for checking for typos 👀🤔😂